doc-content/shared-kie-docs/src/main/asciidoc/KIE/Overview-section.adoc

```diff
@@ -2,5 +2,21 @@
 = Overview
 
 include::Overview/KIEAnatomy-section.adoc[leveloffset=+1]
+
 include::Overview/KIELifeCycles-section.adoc[leveloffset=+1]
 
+// The following is temporarily conditionalized. After the content is made applicable to jBPM by incorporating process-specific and jBPM-specific considerations, we can unconditionalize and make it genuinely shared. (Stetson R., 6 Feb 2019)
```

Comment:
Sure. I usually do for things that need to mass-searched later. Things like this I don't think to tag because it only matters for subsequent times someone wants to update a file. But no losses and only gains for tagging comments. So will do so going forward. Maybe just `@note`.